Build.ps1

```diff
@@ -11,7 +11,7 @@ $suffix = @{ $true = ""; $false = "$branch-$revision"}[$branch -eq "master" -and
 foreach ($src in ls src/Serilog.*) {
     Push-Location $src
 
-    & dotnet pack -c Release -o ..\..\.\artifacts --version-suffix=$suffix
+    & dotnet pack -c Release -o ..\..\.\artifacts --version-suffix=$suffix --include-source
```

Comment:
Yes, this should enable source stepping when "source server support" is enabled in VS.
https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages